website/content/en/preview/contributing/development-guide.md

```diff
@@ -61,6 +61,24 @@ make presubmit # run codegen, lint, and tests
 
 If you are only interested in building the Karpenter images and not deploying the updated release to your cluster immediately with Helm, you can run
 
+*Note: that this will produce a build with the latest version of https://github.com/kubernetes-sigs/karpenter.
+
+```bash
+export $
+make image # build and push the karpenter images
+```
+
+You can test out changes made in https://github.com/kubernetes-sigs/karpenter by replacing the dependency of https://github.com/aws/karpenter-provider-aws/.
+For local changes, replace `$PATH_TO_KUBERNETES_SIGS_KARPENTER` with the relative or absolute path and run:
+
+```bash
+go mod edit -replace sign.k8s.io/karpenter=$PATH_TO_KUBERNETES_SIGS_KARPENTER
+```
+
+*Note: you need to commit the go.mod changes before running `make image`
+
+Then you can build your image as before:
+
 ```bash
 make image # build and push the karpenter images
 ```
```

Comment:
I think you can probably remove these lines now that you've added more detail above